static/js/calculate_score.js

```diff
@@ -20,12 +20,19 @@ function updateResultBoxAndStoredValue(str) {
     resultJSON = str
 }
 
+/**
+ * Currently unused. Sets the the result box of the Calculate page to str and updates the resultJSON to the new text
```

Comment:
Why is this currently unused?

Reply:
Not sure. It looks like it was useful at some point, but isn't called anywhere. I just deleted it for now.